packages/beacon-node/src/chain/bls/multithread/jobItem.ts

```diff
@@ -49,17 +50,31 @@ export function jobItemSigSets(job: JobQueueItem): number {
  * Prepare BlsWorkReq from JobQueueItem
  * WARNING: May throw with untrusted user input
  */
-export function jobItemWorkReq(job: JobQueueItem, format: PointFormat, metrics: Metrics | null): BlsWorkReq {
+export function jobItemWorkReq(
+  deserialized: boolean,
```

Comment:
why `deserialized` not used for `JobQueueItemType.sameMessage`?

Reply:
that is a 🐞...

Reply:
fixed